promptops/promptops/skill_export.py

```
@@ -57,42 +63,127 @@ def generate_skill_content(data: Dict[str, Any], raw_data: Dict[str, Any], raw_c
     
     for path in iter_prompt_files(str(prompts_path)):
-        data = load_yaml(str(path))
-        if not data:
+        errors = []
+        raw_content = ""
+        raw_data = None
+        
+        try:
+            raw_content = path.read_text(encoding='utf-8')
+        except Exception as e:
             continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)